src/redux/books/books.js

```diff
@@ -0,0 +1,48 @@
+const ADD_BOOK = 'bookstore/src/redux/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/src/redux/REMOVE_BOOK';
```

Comment:
- [ ] Kindly be noticed that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`.